server/src/main/java/com/tencent/rss/server/LocalStorageChecker.java

```diff
@@ -62,7 +66,11 @@ public LocalStorageChecker(ShuffleServerConf conf) {
   public boolean checkIsHealthy() {
     int num = 0;
     for (StorageInfo storageInfo : storageInfos) {
-      if (storageInfo.checkIsHealthy()) {
+      if (!storageInfo.checkStorageReadAndWrite()) {
```

Comment:
if there has problem in checkStorageReadAndWrite() and the storage is marked incorrect.
There should be a way to correct this, eg, 
```
if (!checkStorageReadAndWrite) {
  mark false
} else {
  mark true
}
```

Reply:
Not good. If the disk have some bad tracks, these disk will be marked healthy, and then be unhealthy, and then be healthy....

Reply:
ok, it's better to have a command line tool to do such things by manually, and it can be discussed later with a new PR